Delicious Catering/Business Logic Layer/Adult_Menu_Cost.java

```diff
@@ -1,11 +1,10 @@
-public class Adult_Menu_Cost extends Menu_Cost{
-
+public class Adult_Menu_Cost extends Menu_Cost
+{
     @Override
-    public double CalculatMenuCost(int Quantity, String option){
-
-
-        switch (option) {
-
+    public double CalculatMenuCost(int Quantity, String option)
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.